index.html

```diff
@@ -153,6 +154,71 @@ <h1 class="title">Skills</h1>
       </div>
     </div>
   </div>
+  <h1 class="title"> Projects </h1>
+  <div class="projects">
+    <div class="bodycards">
```

Comment:
Why do you need this nested div?.
This should be a single div. Review how to improve the css classes.